packages/edgeconsent/README.md

```diff
@@ -1,4 +1,85 @@
 
-# React Native AEP Consent for Edge Network Extension
+# React Native Consent for Edge Network Extension
 
-// todo
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepedgeconsent.svg)](https://www.npmjs.com/package/@adobe/react-native-aepedgeconsent) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepedgeconsent)](https://www.npmjs.com/package/@adobe/react-native-aepedgeconsent)
+
+`@adobe/react-native-aepedgeconsent` is a wrapper for the iOS and Android [Consent for Edge Network extension](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network) to allow for integration with React Native applications.
+
+## Prerequisites
+
+The Consent for Edge Network extension has the following peer dependency, which must be installed prior to installing the Consent extension:
+- [Core](../core/README.md)
+
+## Installation
+
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page. 
+
+Install the `@adobe/react-native-aepedgeconsent` package:
+
+```bash
+cd MyReactApp
+npm install @adobe/react-native-aepedgeconsent
+```
+
+## Usage
+
+### Installing and registering the extension with the AEP Mobile Core
+Install the Consent extension in your mobile property and configure the default consent preferences by following the steps in the [Consent for Edge Network extension documentation](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network).
+
```

Comment:
When we point the user to this doc, react user might not know they should use the Objective C setup instead of swift.  We should point that out to use the Objective-C setup.

Reply:
👍  Good point, we will clarify that in the GitBook page soon.